### PR TITLE
Rebuilds with mold 1.5.1 to fix breakage from mold 1.5.0

### PR DESCRIPTION
**Pull request by satmandu**
- still waiting on rebuild of cmake to complete...

## Works properly:
- [x] `x86_64`
- [x] `i686`
- [x] `armv7l`

### Run the following to get this pull request's changes locally for testing.
```
CREW_TESTING_REPO=https://github.com/satmandu/chromebrew.git CREW_TESTING_BRANCH=mold_rebuild CREW_TESTING=1 crew update
```

### CODE REVIEW

File: packages/ninja.rb [hunk 1]
```diff
@@ -3,23 +3,23 @@
...
   description 'a small build system with a focus on speed'
   homepage 'https://ninja-build.org'
-  version '1.11.1'
+  version '1.11.1-1'
   license 'GPL-2'
   compatibility 'all'
-  source_url 'https://github.com/ninja-build/ninja/archive/v1.11.1.tar.gz'
-  source_sha256 '31747ae633213f1eda3842686f83c2aa1412e0f5691d1c14dbbcc67fe7400cea'
+  source_url 'https://github.com/ninja-build/ninja.git'
+  git_hashtag 'v1.11.1'
```
Comment (uberhacker) [inline packages/ninja.rb:10]: Use `@_ver`?

Comment (satmandu) [reply packages/ninja.rb:10]: done.

Comment (uberhacker) [reply packages/ninja.rb:10]: I'm not seeing a change.  Did I miss it?

Comment (satmandu) [reply packages/ninja.rb]: one git push later...

**Review by uberhacker — APPROVED** [review]

**Pull request merged**